### DISCUSSION TITLE
Blur wallpaper on window focus

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [hyprwm/Hyprland#2090] [Q&A] [answered]
https://github.com/bouteillerAlan/blurredwallpaper
Is there anything similar for hyprland?

**Comment by vaxerski** [comment, marked as answer]
no sir, but you can make one yourself

**Reply by LamprosPitsillos** [reply]
Great! Is this possible using the plug-in system?

**Reply by vaxerski** [reply]
si

**Reply by BeyondMagic** [reply]
@LamprosPitsillos did you make?
